views/templates/admin/backend.tpl

```diff
@@ -16,11 +16,33 @@
  * @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
  * International Registered Trademark & Property of Blockonomics
  *}
-<div class="alert alert-info">
-<p><strong>{l s="You are few clicks away from accepting bitcoin payments" mod='blockonomics'}</strong></p>
-</br>
-<p>Click on <strong>Get Started for Free</strong> on <strong><a
-href="https://www.blockonomics.co/merchants" target="_blank">Blockonomics
-Merchants</a></strong>,
-complete the wizard and copy the API Key when shown here</p>
-</div>
+{if $temp_api_key && !$api_key && !($total_received > 0)}
+    <div class="alert alert-info">
+    	<p><b>Blockonomics Wallet</b> (Balance: 0 BTC)</p>
+        <p>We are using a temporary wallet on Blockonomics to receive your payments.</p>
+        <p>
+        	To receive payments directly to your wallet (recommended) -> Follow Wizard by clicking on <i>Get Started for Free</i> on <a href="https://www.blockonomics.co/merchants" target="_blank">Merchants</a> and enter the APIKey below [<a href="https://blog.blockonomics.co/how-to-accept-bitcoin-payments-on-woocommerce-using-blockonomics-f18661819a62">Blog Instructions</a>]
+        </p>
+    </div>
+{elseif $temp_api_key && $total_received > 0}
+	<div class="alert alert-info"><p><b>Blockonomics Wallet</b> (Balance: {$total_received|escape:'htmlall':'UTF-8'} BTC)</p>
+    {if !$api_key}
+	    <p>
+	    	To withdraw, follow wizard by clicking on <i>Get Started for Free</i> on <a href="https://www.blockonomics.co/merchants" target="_blank">Merchants</a>, then enter the APIKey below [<a href="https://blog.blockonomics.co/how-to-accept-bitcoin-payments-on-woocommerce-using-blockonomics-f18661819a62">Blog Instructions</a>]
+	    </p>
+	{else}
+	    <p> To withdraw, Click on <b>Test Setup</b></p>
+	{/if}
+	</div>
+{elseif $api_key}
+    <div class="alert alert-info">
+    	<p><b>Your wallet</b></p>
+        <p>
+        	Payments will go directly to the wallet which your setup on <a href="https://www.blockonomics.co/merchants" target="_blank">Blockonomics</a>. There is no need for withdraw
```

Comment:
Typo here:

> to the wallet which your setup on

Reply:
Fixed with commits e776326, eaa7b6a
This will also require fixing on woocommerce https://github.com/blockonomics/woocommerce-plugin/blob/master/blockonomics-woocommerce.php#L261